src/main/formats/Vab.cpp

```diff
@@ -233,7 +223,7 @@ bool VabRgn::LoadRgn() {
 
   AddGeneralItem(dwOffset, 1, L"Priority");
   AddGeneralItem(dwOffset + 1, 1, L"Mode (use reverb?)");
-  AddVolume((GetByte(dwOffset + 2) * instr->masterVol) / (127.0 * 127.0), dwOffset + 2, 1);
+  AddVolume(long{(GetByte(dwOffset + 2) * instr->masterVol)} / (127.0 * 127.0), dwOffset + 2, 1);
```

Comment:
I have compiled Vab.cpp in Visual Studio 2022 with the warning level set to maximum, but no warning appears on this line. In my opinion, this code may not overflow even in its original form.

```suggestion
  AddVolume((GetByte(dwOffset + 2) * instr->masterVol) / (127.0 * 127.0), dwOffset + 2, 1);
```